<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1583141286450' stop='1583141292401'>
    <name>API Testing</name>
    <title>API Testing</title>
    <test-cases>
        <test-case start='1583141286452' status='failed' stop='1583141288454'>
            <name>API:GET CALL @smoke</name>
            <title>API:GET CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d3ad4d7a-bda6-4b66-af8d-7dcbb3a23774-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected 200 to equal 201.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:61:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1583141288454' status='failed' stop='1583141290309'>
            <name>API:POST CALL @smoke</name>
            <title>API:POST CALL @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='42a6d881-4cda-4194-ae1e-51f3b9dff12f-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected 'VCA-OU-167' to equal 'VCA-OU-72'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:101:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1583141290309' status='passed' stop='1583141291380'>
            <name>API:PUT CALL @regression</name>
            <title>API:PUT CALL @regression</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='90e5ede1-265a-4c9b-939d-3aa948f521a1-attachment.png' type='image/png' size='3249'/>
            </attachments>
        </test-case>
        <test-case start='1583141291380' status='failed' stop='1583141292400'>
            <name>API:GET CALL FOR NOTES @smoke</name>
            <title>API:GET CALL FOR NOTES @smoke</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='1eb27979-6941-43ec-af80-1b153ee17c80-attachment.png' type='image/png' size='3249'/>
            </attachments>
            <failure>
                <message>Expected Object({ id: 113, accountNumber: 'VCA-OU-72', createdBy: 'Manoj', updatedBy: null, note: 'This not is created' }) to equal 'VCA-OU-72'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (D:\Mars_VoyagerAPI\TestSuites\sampleTestCase.spec.js:137:37)
                    at processTicksAndRejections (internal/process/task_queues.js:97:5)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>